tracing_test.go

```diff
@@ -86,14 +86,9 @@ func (s *TracingSuite) TestParameterLoggingMiddleware() {
 		})(resp, req)
 	fmt.Println(buf)
 	log.SetOutput(os.Stdout)
-	res := map[string]interface{}{}
-	err := json.Unmarshal(buf.Bytes()[5:len(buf.Bytes())-1], &res)
-	s.NoError(err)
-	value, found := res["some-key"]
-	s.True(found)
-	s.Equal("some-value", value.(string))
-	_, found = res["unrelated-stuff"]
-	s.False(found)
+
+	// Should not log anything
+	s.Equal(0, len(buf.Bytes()))
```

Comment:
I would just remove the test entirely if we don't want this functionality anymore